content/config.yml

```diff
@@ -1,223 +1,859 @@
-    video: http://pyvideo.org/video/2637/set-your-code-free-releasing-and-maintaining-an
-
-  - title: 'Susy Grids 2.0'
-    author: miriam
-    date: March 13
-    year: 2014
-    forum: Sass Bites
-    forum_url: https://www.youtube.com/user/sassbites
-    video: https://www.youtube.com/watch?v=m7k-vBzebPg
-
-  - title: 'Python Testing Tools'
-    author: carl
-    date: February 28
-    year: 2014
-    forum: ConFoo
-    forum_url: http://confoo.ca/
-    slides: http://oddbird.net/python-testing-tools-preso/
-
-  - title: 'Advanced Python'
-    author: carl
-    date: February 27
-    year: 2014
-    forum: ConFoo
-    forum_url: http://confoo.ca/
-    slides: http://oddbird.net/advanced-python-preso/
-
-  - title: 'Organizing Sass Partials'
-    author: miriam
-    date: November 19
-    year: 2013
-    forum: NYCSass Meetup
-    forum_url: http://www.meetup.com/nyc-sass/events/146898452/
-    slides: http://miriamsuzanne.com/pres/sass-partials/
-
-  - title: 'Show Your Work <i class="amp">&</i> Share Your Toys'
-    author: miriam
-    date: October 12
-    year: 2013
-    forum: SassConf
-    forum_url: http://sassconf.com/2013/
-    slides: http://miriamsuzanne.com/pres/show-yr-work/index.html
-    video: https://vimeo.com/channels/sassconf2013/86306775
-
-  - title: 'Getting Started with Automated Testing'
-    author: carl
-    date: March 16
-    year: 2013
-    forum: PyCon
-    forum_url: https://us.pycon.org/2013/
-    slides: http://oddbird.net/start-testing-preso/
-    video: http://pyvideo.org/video/1674/getting-started-with-automated-testing
-
-
-# Company
-# -------
-company:
-  - name: OddBird
+  - venue: 'CascadiaFest'
+    url: 'http://2016.cascadiafest.org/'
+    date: ['08', '03', '2016']
+    who:
+      - author: 'miriam'
+        title: 'Code Patterns for Pattern-Making'
+
+  - venue: 'Plone Digital Experience Conference'
+    url: 'http://2016.ploneconf.org/'
+    date: ['10', '19', '2016']
+    who:
+      - author: 'david'
+        title: 'Integration*5'
+
+
+# Client Projects
+# ---------------
+
+clients:
+  - name: 'Quarq Race Intelligence'
+    url: 'http://quarqrace.com'
+    years: '2015-2016'
+    feature: true
+    priority: 'live'
+    tagline: 'Real-Time Race Analytics'
+    description: |
+      Quarq Race Intelligence captures and interprets
+      race performance and state-of-play information,
+      and delivers that data in real time to spectators,
+      commentators, race officials,
+      third-party analysis software and visualization tools.
+      Integrating with Quarq’s telemetry and reporting hardware,
+      the app shows racers’ positions, direction, speed, heart-rate,
+      and a number of other statistics in real time.
+      Our mobile-optimized web interface has been used for bicycle races,
+      triathlons, and non-competitive events,
+      including regular IronMan and Enduro races.
+    client:
+      name: 'SRAM'
+      url: 'http://sram.com/'
+
+  - name: 'QuarqNet'
```

Comment:
This isn't an NDA situation, just not ready to advertise it broadly on our site.